charts/argocd-apps/ci/staging-values.yaml

```diff
@@ -0,0 +1 @@
+../values-staging.yaml
```

Comment:
why do we need this relative path?

Reply:
It's a symlink. [chart-testing](https://github.com/helm/chart-testing) is a bit inflexible about which values files it tests against; it forces you to put the files to test against in a directory called `ci` and only matches filenames of the pattern `*-values.yaml`. So in order to get it to test that the values files work with the chart, we have to symlink them here. Maybe someday we can upstream a fix so that we can just tell chart-testing which values files to use.